tools/donate_cpu_lib.py

```diff
@@ -269,12 +269,15 @@ def scan_package(work_path, cppcheck_path, jobs, libraries):
     libs = ''
     for library in libraries:
         if os.path.exists(os.path.join(cppcheck_path, 'cfg', library + '.cfg')):
-            libs += ' --library=' + library
+            libs += '--library=' + library + ' '
 
     # Reference for GNU C: https://gcc.gnu.org/onlinedocs/cpp/Common-Predefined-Macros.html
-    options = jobs + libs + ' --showtime=top5 --check-library --inconclusive --enable=style,information --template=daca2 -rp=temp'
-    if os.path.isfile('temp/TortoiseSVN.sln'):
-        options += ' --platform=win64 --project=temp/TortoiseSVN.sln'
+    options = libs + jobs + ' --showtime=top5 --check-library --inconclusive --enable=style,information --template=daca2 -rp=temp'
+    if os.path.isfile('temp/tortoisesvn/TortoiseSVN.sln'):
+        options = options.replace('--library=posix ', '')
```

Comment:
it's a bit unfortunate that `options` contain this at all so it must be removed.. but well we can fix that later.

Reply:
Yes, that should be refactored.
If more Windows projects will be added this should be changed.
Did you add TortoiseSVN just for testing or for preparing the client for Windows projects or so?